WebContent/WEB-INF/jsp/collectionImport.jsp

```diff
@@ -73,7 +64,7 @@
             }
         }
         
-        var message = "Are you sure you want to Create collection '" + collTitle + "'? \n";
+        var message = "Are you sure you want to Create new collection? \n";
```

Comment:
@mcritchlow It's done. See commit https://github.com/ucsdlib/damsmanager/pull/159/commits/76e24a8ea8e34d4300900b2499de2ffad654d1a9. Thanks.